WTWSMS/events/religious_events.txt

```diff
@@ -3763,11 +3809,16 @@ character_event = {
 			
 			# Pragmatism
 			modifier = {
-				factor = 3  # Powerful rulers want a modern religion
+				factor = 4  # Powerful rulers want a modern religion
 				tier = KING
 				realm_size = 12
 			}
 			modifier = {
+				factor = 6
+				tier = EMPEROR
+				realm_size = 24
+			}
+			modifier = {
```

Comment:
Dude, I was giving an example. When a Khagan SETTLES DOWN, he gets a FEUDAL Empire that can often be an OPM. He should not be more likely to convert simply because he has a mock Empire.